bin/ares-package.js

```diff
@@ -73,6 +75,8 @@ function PalmPackage() {
         "pi":       "--pkgid",
         "pv":       "--pkgversion",
         "pf":       "--pkginfofile",
+        "i":        "--info",
+        "id":       "--info-detail",
```

Comment:
How about -I instead of "-id" ? 
ares-setup-device also has '"-F" option for listFull.

Reply:
I change this to "I"